src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -31,14 +32,18 @@
     private final String telegram;
     private final String studentId;
     private final List<JsonAdaptedTag> tagged = new ArrayList<>();
+    private final List<JsonAdaptedLabNumber> labNumbers = new ArrayList<>();
+    private final List<JsonAdaptedLabStatus> labStatuses = new ArrayList<>();
```

Comment:
Why cant these two lines be just a <code>List\<JsonAdaptedLab\></code>? That way we don't need <code>JsonAdaptedLabNumber</code> and <code>JsonAdaptedLabStatus</code> also, will just need <code>JsonAdaptedLab</code>?

Reply:
Oh, right that was what I was originally going for. I wanted to just have `JsonAdaptedLab` but ran into some difficulty implementing it so I split them up for now as I wanted to push this commit so that the group can continue working on lab-related tasks. As mentioned to @channne during our last project meeting, I require some help with this so maybe you could try working on this part.